.eslintrc.cjs

```diff
@@ -32,6 +37,7 @@ module.exports = {
         objectLiteralTypeAssertions: 'allow'
       }
     ],
+    '@typescript-eslint/no-empty-function': 'off',
```

Comment:
There is only one instance of an empty function in the tests. I suggest modifying the test instead of adding a rule that affects the entire source code.

Reply:
OK, after discussion I fine with that rule

Reply:
Discussed offline